### DISCUSSION TITLE
Individual software cannot be opened in sandbox

### DISCUSSION THREAD

**Discussion by 348444193** [sandboxie-plus/Sandboxie#3171] [Q&A Feedback]
Individual software cannot be opened in the latest version of the sandbox (version number 5.65.2), such as excel2013 and QQ browser, my operating system version is window10 22H2, and the kernel version is 19045. 
Can it be adapted?
Thank you!

**Comment by isaak654** [comment]
1. https://github.com/sandboxie-plus/Sandboxie/discussions/2758#discussioncomment-5425193
2. [CONTRIBUTING.md#how-to-help-with-sandboxie-issues](https://github.com/sandboxie-plus/Sandboxie/blob/master/CONTRIBUTING.md#how-to-help-with-sandboxie-issues)
